### ISSUE TITLE
Basic types failed to be handled in object definition

### ISSUE THREAD

**Issue by brchiu** [vlm/asn1c#147]
After applying #115 or use [mouse07410's repository](https://github.com/mouse07410/asn1c) having APER-capability, when processing ASN.1 file of 3GPP's S1AP (36.413) by asn1c, fatal error message occurred at `RerouteNASRequest-IEs` :

```
FATAL: Cannot find OCTET referenced by RerouteNASRequest-IEs at line 1545 in ./S1AP-PDU-Contents.asn
FATAL: Parsing ObjectSet RerouteNASRequest-IEs failed at 1545 in ./S1AP-PDU-Contents.asn
```
The content of `RerouteNASRequest-IEs` is :

```
RerouteNASRequest-IEs S1AP-PROTOCOL-IES ::= {
	{ ID id-eNB-UE-S1AP-ID			CRITICALITY reject	TYPE ENB-UE-S1AP-ID			PRESENCE mandatory}|
	{ ID id-MME-UE-S1AP-ID			CRITICALITY ignore	TYPE MME-UE-S1AP-ID			PRESENCE optional}|
	{ ID id-S1-Message			CRITICALITY reject	TYPE OCTET STRING			PRESENCE mandatory}|
	{ ID id-MME-Group-ID			CRITICALITY reject	TYPE MME-Group-ID			PRESENCE mandatory}|
	{ ID id-Additional-GUTI			CRITICALITY ignore	TYPE Additional-GUTI		PRESENCE optional},
	...
}
```
This issue comes from :
1. function `_asn1f_parse_class_object_data` does not handle field name having space within it, so only `OCTECT` of `OCTET STRING` is read.
2. function `asn1f_lookup_symbol`, called in `_asn1f_assign_cell_value` does not handle basic type like `OCTECT STRING`, `INTEGER` .... etc.

Part one, for the specific `OCTET STRING` case, can be fixed without much difficulty.
Part two, might need some more modification.

**Comment by mouse07410** [comment]
@brchiu care to submit a PR (with test-cases) for (1) to my fork? :-)
There aren't that many of multi-word types, I think: `OCTET STRING`, `SEQUENCE OF`, `SET OF` - anything else?

**Comment by brchiu** [comment]
This issue is solved with better approach in [pr 22 of mouse07410 repository](https://github.com/mouse07410/asn1c/pull/22). 

A separate pull request to this main repository is to be prepared.

**Comment by brchiu** [comment]
Issue fixed after merged pr #154 and checked ok on newest commit 6aae7c62c1883e007108e79d3123e7b17940da11 .

Issue closed.
